### ISSUE TITLE
Missing code coloring on slide

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#516]
```
Please describe the issue:
This slide: http://slides.html5rocks.com/#new-form-types 
has the attribute 'placeholder' colored red, two out of three times.

Please provide any additional information below.
Perhaps an idea to add the <b></b> tags around the other one as well?
```

Original issue reported on code.google.com by `rijstbal...@gmail.com` on 3 May 2011 at 9:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 3 May 2011 at 9:57
- Changed state: **Assigned**
- Added labels: **Milestone-6**, **Slides**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision 1a5fb41e03c2 
(http://code.google.com/p/html5rocks/source/detail?r=1a5fb41e03c280a0c697c1b2aa0
a45fd270e5a1e)
```

Original comment by `han...@google.com` on 4 May 2011 at 4:15
- Changed state: **Fixed**
